css/style.css

```diff
@@ -0,0 +1,225 @@
+  margin-bottom: -5px;
+}
+
+#header-flex #header-link:hover {
+  text-decoration: underline;
+}
+
+/* HEADER */
+
+/* NAVBAR */
+
+#nav-position {
+  position: relative;
+  top: 83px;
+  height: 36px;
+  width: 980px;
+  margin: 0 auto;
+}
+
+#nav-position #nav-container {
+  display: grid;
+  grid-template-columns: 102px repeat(7, 105px) 1fr;
+}
+
+#nav-container .bg-nav {
+  background-image: linear-gradient(180deg, rgba(115, 115, 115, 1) 0%, rgba(94, 94, 94, 1) 50%, rgba(112, 112, 112, 1) 51%, rgba(141, 141, 141, 1) 100%);
+}
+
+#nav-container .nav-b {
+  height: 36px;
+  width: 100%;
+  border-top: 1px solid #668;
+  border-right: 1px solid #484848;
+  border-bottom: 1px solid #668;
+  color: white;
+}
+
+#nav-container .nav-b:hover {
+  background: linear-gradient(180deg, rgba(78, 78, 78, 1) 0%, rgba(55, 55, 55, 1) 50%, rgba(75, 75, 75, 1) 51%, rgba(98, 98, 98, 1) 100%);
+}
+
+#nav-container .nav-b:first-child {
+  border-radius: 5px 0 0 5px;
+}
+
+#nav-container .input-b {
+  height: 36px;
+  width: 100%;
+  border-radius: 0 5px 5px 0;
+}
+
+.nav-b #small-logo {
+  height: 20px;
+  width: 20px;
+}
+
+#input-nav input {
+  height: 18px;
+  width: 105px;
+  border: 1px solid #2f2f2f;
+  border-radius: 15px;
+  margin: 7px 13.5px;
+  background: transparent;
+  padding-left: 31px;
+}
+
+.material-icons {
+  position: absolute;
+  top: 8px;
+  right: 100px;
+  filter: invert(100%);
+}
+
+/* NAVBAR */
+
+/* ASIDE */
+
+#aside-container {
+  display: flex;
+  flex-direction: row;
+  height: 200px;
+  width: 100%;
+}
+
+#aside-container .promo-img {
+  height: 200px;
+  width: 337px;
+}
+
+#aside-container .promo1 {
+  background: url(https://web.archive.org/web/20140301004349im_/http://images.apple.com/home/images/promo_iphone5s.jpg);
+}
+
+#aside-container .promo2 {
+  background: url(https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c.jpg);
+}
+
+#aside-container .promo3 {
+  background: url(https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years.jpg);
+}
+
+#aside-container .promo4 {
+  background: url(https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility.jpg);
+}
+
+/* ASIDE */
+
+/* FOOTER */
+
+#footer-container {
+  height: 45px;
+  width: 980px;
+  margin: 30px auto 25px;
+  font-family: Arial, 'Lucida Grande', 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Sans Unicode', sans-serif;
+  font-size: 10px;
+  font-stretch: ultra-condensed;
+  color: #6e6e6e;
+}
+
+#footer-container .footer-items {
+  float: left;
+  height: 27px;
+  width: 50%;
+}
+
+.footer-items .item {
+  float: left;
+  height: auto;
+  border-left: 1px solid #ccc;
+  padding-left: 4px;
+  margin-left: 4px;
+}
+
+.item a:hover {
+  text-decoration: underline;
+}
+
+.item:first-child {
+  border-left: 0;
+  padding-left: 0;
+  margin-left: 0;
+}
+
+.item:nth-child(8) {
+  border-left: 0;
+  padding-left: 0;
+  margin-left: 0;
+}
```

Comment:
#### CSS Bad practices

- [ ] instead of having 2 elements with the same styling you can just do this:

.item:first-child,
.item:nth-child(8) {
  border-left: 0;
  padding-left: 0;
  margin-left: 0;
}